### PR TITLE
WTAZ open LC 13

### PR DESCRIPTION
**Pull request by mnaguit**
- borehole sensor: TH120PH-2 (9205)
- depth (90m) & orientation (36)
- For LC 13 (HH/LH-12Z) streams

Can you please double check the sensor details for this one? Thanks @CBurton90

### CODE REVIEW

**Comment by CBurton90** [comment]
@mnaguit `TrilliumCompact 9205` (Trillium Compact 120PH-2) is correct

**Review by CBurton90 — APPROVED** [review]

File: install/sensors.csv
Comment (salichon) [inline install/sensors.csv:1]: Need sensor borehole  loc 13  orientation to  ~36 or 39 degrees
thanks

**Review by salichon — CHANGES_REQUESTED** [review]
WTAZ HH 13 azimuth :)

File: install/sensors.csv
Comment (mnaguit) [reply install/sensors.csv:1]: Yay. Thanks @salichon . Entry has been modified for azimuth.

**Comment by mnaguit** [comment]
Another round of review, please Gentlemen! 
Thanks!

**Review by CBurton90 — APPROVED** [review]

**Review by salichon — APPROVED** [review]
ouioui

**Pull request merged**